src/components/HowItWorksSection.tsx

```diff
@@ -0,0 +1,96 @@
+import Image, { StaticImageData } from "next/image";
+import Step1Button from "../../public/step1-button.png";
+import Step2Describe from "../../public/step2-describe-chat.png";
+import Step3Answer from "../../public/step3-answer.png";
+
+interface StepCardProps {
+    stepNumber: number;
+    title: string;
+    description: string;
+    image: StaticImageData;
+    reverse?: boolean;
+    width?: number;
+    height?: number;
+}
+
+function StepCard({
```

Comment:
I think it's just a code style/preference thing, there's not really a rule to have 1 component per file. In this case it seems fine to me since `StepCard` won't be used anywhere else in the project. It's the same approach i did in my `DetailsSection` component, imo it just looks neat and helps keep the code tidy